### ISSUE TITLE
Strange behavior

### ISSUE THREAD

**Issue by vangelov** [bee-queue/bee-queue#66]
Hello,

I'm getting this super strange behavior:

There are 2 files 

```javascript
// file1.js:

var Queue = require('bee-queue');
var queue = new Queue('test-queue');

var job = queue.createJob();

job.on('succeeded', function (result) {
	console.log('succeeded');
});

job.on('failed', (err) => {
	console.log('failed', err);
});

job.save();
```

```javascript
// file2.js:

var Queue = require('bee-queue');
var queue = new Queue('test-queue');

queue.process(function (job, done) {
  done(null);
});
```

First I run `node file2.js`.

Then, when I run `node file1.js`, the `succeeded` callback is called, which is correct.
But if I run `node file1.js` again, neither the `succeeded`, nor `failed` event is emitted.

Here'a link to a sample project: https://dl.dropboxusercontent.com/u/25695112/project.zip

Am I doing something wrong?

**Comment by skeggse** [comment]
I'm not able to repro with the given snippets, and I'm unfortunately not willing to download a zip (though I'd look at a GitHub repo or Gist if provided). It's also possible that whatever issue you're seeing has been fixed as of 1.0.0.

If you continue seeing this issue, feel free to reopen. If that's the case, would you please include your Node.js version, npm version (or indicate that you're installing via Git), Redis version, and operating system version?

**Comment by vangelov** [comment]
Just upgraded to 1.1.0 and the issue seems to be fixed. Thanks!
